docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -176,53 +176,49 @@ query {
 If you skipped the last step and want to use frontmatter for your
 slugs instead of the generated field, replace `fields` with `frontmatter`.
 
-```javascript=gatsby-node.js
+```javascript:title=gatsby-node.js
 const path = require("path")
 
-exports.createPages = ({ graphql, actions }) => {
+exports.createPages = async ({ graphql, actions, reporter }) => {
   // Destructure the createPage function from the actions object
   const { createPage } = actions
 
-  return new Promise((resolve, reject) => {
-    resolve(
-      graphql(
-        `
-          {
-            allMdx {
-              edges {
-                node {
-                  id
-                  fields {
-                    slug
-                  }
-                }
-              }
+  const result = await graphql(`
+    query {
+      allMdx {
+        edges {
+          node {
+            id
+            fields {
+              slug
             }
+            body
```

Comment:
```suggestion
```

`body` isn't used in the gatsby-node file so doesn't need to be queried here.

Reply:
Thank you for your feedback. Line 195 remove.